aio/content/navigation.json

```
@@ -102,6 +102,11 @@
       "title": "Understanding Angular",
       "tooltip": "Learn the concepts essential to becoming a proficient Angular developer.",
       "children": [
+        {
+          "title": "Overview",
+          "url": "guide/understanding-angular-overview",
+          "tooltip": "Learn about the most common elements of Angular applications."
+        },
         {
           "title": "Components",
           "tooltip": "Building dynamic views with data binding",
```

```suggestion
          "tooltip": "Learn more about how to use data binding to build dynamic views",
```

Changed to match similar usage.  

Changed phrase to single target.